File: partie_extras/p40_R_vs_python/analyze.py
import pandas as pd
import subprocess

# Lecture et traitement (ex. ici juste affichage pour la démo)
df = pd.read_csv("data.csv")
print("Données analysées en Python :")
print(df.describe())

# Appel du script R pour le graphique
subprocess.run(["Rscript", "plot.R"])

File: partie_extras/p40_R_vs_python/analyze.R
# Appel du script Python
system("python3 preprocess.py")

# Analyse des données nettoyées
data <- read.csv("data_clean.csv")
summary(data)

plot(data$x, data$y_norm, type="b", col="darkgreen", pch=16,
     main="Analyse R après traitement Python",
     xlab="x", ylab="y normalisé")
